spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/deployer/ModuleDeployer.java

```diff
@@ -38,6 +38,8 @@
 
 	public static final String JMX_DEFAULT_DOMAIN_KEY  = "spring.jmx.default-domain";
 
+	public static final String MODULE_LAUNCHER_PROPERTY_PREFIX = "launcher.";
```

Comment:
Add some javadoc to explain how this is being used (e.g. with a concrete example)